Fix value of lookup sync period in docs (#13695)
apache/druid: docs/configuration/index.md

```

##### Lookups Dynamic Configuration

> These configuration options control the lookup management done by the coordinator. Configurations that affect propagation of lookups can be found in [dynamic configuration for lookups](../querying/lookups.md#dynamic-configuration).

Reviewer: ```suggestion
These configuration options control Coordinator lookup management. See [dynamic configuration for lookups](../querying/lookups.md#dynamic-configuration) configurations that affect lookup propagation.
```

|`druid.manager.lookups.deleteAllTimeout`|How long to wait for all `DELETE` requests to finish before considering the delete attempt a failure|PT10S|
|`druid.manager.lookups.updateAllTimeout`|How long to wait for all `POST` requests to finish before considering the attempt a failure|PT60S|
|`druid.manager.lookups.threadPoolSize`|How many processes can be managed concurrently (concurrent POST and DELETE requests). Requests this limit will wait in a queue until a slot becomes available.|10|

> |`druid.manager.lookups.period`|How many milliseconds between checks for configuration changes|120000 (2 mins)|

Reviewer: ```suggestion
|`druid.manager.lookups.period`|Number of milliseconds between checks for configuration changes|120000 (2 minutes)|
```
